### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/feature/encrypt.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#544] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/feature/encrypt.html

主题支持对特定文件夹或特定的路径进行加密，也支持进行全局范围的加密。 警告 注意，受到 VuePress 的限制，在未解密前，文章内容仅仅被隐藏，访问者仍可以从源码中获取文章的内容。 所以请不要使用该加密功能用于任何敏感、机密的文章与档案，造成的后果请你自负。

https://theme-hope.vuejs.press/zh/guide/feature/encrypt.html

**Comment by zuihou** [comment]
是否支持在弹出的 数据密码框中自定义标题或说明文案？

（比如：在密码框上方告诉用户如何获取密码等文案）

**Reply by w1ndys** [reply]
同问，蹲

**Reply by Mister-Hope** [reply]
新版本将会支持。https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/07ae433e4c622d6fc07b8b3e79e35d8653b70d2c

**Reply by w1ndys** [reply]
非常感谢  @Mister-Hope

**Comment by w1ndys** [comment]
是否支持在加密某个目录下，选择这个目录某个文件不进行加密，例如加密a目录下所有文件，但不加密a目录下的b文件

**Reply by Mister-Hope** [reply]
目前不支持，不过可以考虑引入正则。

**Reply by w1ndys** [reply]
谢谢，我试一下

**Reply by w1ndys** [reply]
@Mister-Hope  下面这个正则部署后没有效果，现象是该目录都没有被加密

"^/EasySelectCourse/CourseSelectionRecommendation/(?!Contributors\\.html$).*$":

理想情况下是加密/EasySelectCourse/CourseSelectionRecommendation/，不加密/EasySelectCourse/CourseSelectionRecommendation/Contributors.html

**Reply by Mister-Hope** [reply]
我只是说我会考虑一下

**Reply by w1ndys** [reply]
> 我只是说我会考虑一下

哈哈好的，期待
